opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/InfoDTO.java

```diff
@@ -42,6 +44,7 @@
     private String packageDescription;
     private TicketerConfig ticketerConfig;
     private DatetimeformatConfig datetimeformatConfig;
+    private Map<String,String> services;
```

Comment:
I always like to initialize with empty map. Guess current code path doesn't have any problem but to be safe.